lib/matplotlib/backends/backend_pdf.py

```diff
@@ -683,14 +682,14 @@ def __init__(self, filename, metadata=None):
         self._soft_mask_seq = (Name(f'SM{i}') for i in itertools.count(1))
         self._soft_mask_groups = []
         # reproducible writeHatches needs an ordered dict:
```

Comment:
dunno if you want to update the comment here and below

Reply:
We can just remove it.